resources/zib/zib-TreatmentObjective.xml

```diff
@@ -0,0 +1,213 @@
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-treatmentobjective-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/TreatmentObjective-v3.2(2020EN)" />
+    <name value="zib TreatmentObjective-v3.2(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-functionalormentalstatus-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/FunctionalOrMentalStatus-v3.2(2020EN)" />
+    <name value="zib FunctionalOrMentalStatus-v3.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Goal" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Goal" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Goal">
+      <path value="Goal" />
+      <short value="TreatmentObjective" />
+      <comment value="Although the zib model assumes there's a reference from zib OutcomeOfCare to a distinct instance of zib FunctionalOrMentalStatus (in future context), in this profile the concepts of zib FunctionalOrMentalStatus are mapped directly as this aligns with the way the Goal resource is used." />
+      <alias value="Behandeldoel" />
+      <mapping>
+        <identity value="zib-treatmentobjective-v3.2-2020EN" />
+        <map value="NL-CM:13.5.1" />
+        <comment value="TreatmentObjective" />
+      </mapping>
+    </element>
+    <element id="Goal.lifecycleStatus">
+      <path value="Goal.lifecycleStatus" />
+      <comment value="No zib concept can be mapped (implicitly). Unless `lifecycleStatus` is explicitly recorded, TreatmentObjectives may assumed to be _active_." />
+    </element>
+    <element id="Goal.description.text">
+      <path value="Goal.description.text" />
+      <short value="DesiredHealthcareResult / StatusName / StatusValue" />
+      <definition value="The textual account of the treatment objective. This concept provides the option to define a treatment objective which cannot be described as a measurement value or functional/mental status." />
+      <comment value="If the concept DesiredHealthcareResult (NL-CM:13.5.5) is not present in this instance of zib TreatmentObject, but DesiredHealthCondition (NL-CM:13.5.3) is, `description.text` should be filled with a textual representation of FunctionalOrMentalStatus.StatusName and FunctionalOrMentalStatus.StatusValue." />
+      <alias value="GewenstZorgresultaat" />
+      <mapping>
+        <identity value="zib-treatmentobjective-v3.2-2020EN" />
+        <map value="NL-CM:13.5.5" />
+        <comment value="DesiredHealthcareResult" />
+      </mapping>
+      <mapping>
+        <identity value="zib-functionalormentalstatus-v3.2-2020EN" />
+        <map value="NL-CM:4.26.2" />
+        <comment value="StatusName (implicit, main mapping is on Goal.target.measure)" />

```

Comment:
Should we add quotes to 'Goal.target.measure' (similarly in line 71)?

Reply:
Backticks to be precise. According to the profiling guidelines, we don't do that (it's not Markdown)

Reply:
`.mapping.comment` is indeed not Markdown but a string. I will resolve this conversation.